src/core/network/createResponse.js

```diff
@@ -13,37 +13,45 @@ governing permissions and limitations under the License.
 import { flatMap } from "../../utils";
 
 /**
- * Represents a gateway response with the addition to helper methods.
- *
- * @param {Object} respDto The raw JSON response from the edge gateway
- *  - Current schema:
- * {
- *      requestId: {String},
- *      handle: [
- *          { type, payload }
- *      ]
- * }
- *
- * @returns {Object<Response>} A Response object containing:
- *  - `getPayloadsByType`: returns matching fragments of the response by type
- *      - @param {String} type: A string with the current format: <namespace:action>
- *          example: "identity:persist"
+ * Creates a representation of a gateway response with the addition of
+ * helper methods.
+ * @returns Response
  */
-export default (content = { requestId: "", handle: [] }) => {
-  // TODO: Should we freeze the response to prevent change by Components?
-  // Object.freeze(response.handle.map(h => Object.freeze(h)));
+export default (content = {}) => {
+  const { handle = [], errors = [], warnings = [] } = content;
```

Comment:
Is it weird that it's `handle` and not `handles`? I think it's like that because we are matching the Konductor response... :)

Reply:
"handle" and "handles" are both a little odd to me. Something like "actions" would seem more appropriate I think, but I probably prefer consistency with Konductor over being right here. 😄

Reply:
I agree 🤐  :)